tests/countries/test_bhutan.py

```diff
@@ -0,0 +1,152 @@
+#  holidays
+#  --------
+#  A fast, efficient Python library for generating country, province and state
+#  specific sets of holidays on the fly. It aims to make determining whether a
+#  specific date is a holiday as fast and flexible as possible.
+#
+#  Authors: Vacanza Team and individual contributors (see CONTRIBUTORS file)
+#           dr-prodigy <dr.prodigy.github@gmail.com> (c) 2017-2023
+#           ryanss <ryanssdev@icloud.com> (c) 2014-2017
+#  Website: https://github.com/vacanza/holidays
+#  License: MIT (see LICENSE file)
+
+from unittest import TestCase
+
+from holidays.countries.bhutan import Bhutan, BT, BTN
+from tests.common import CommonCountryTests
+
+
+class TestBhutan(CommonCountryTests, TestCase):
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass(Bhutan, years=(range(1970, 2050)))
+
+    def test_country_aliases(self):
+        self.assertAliases(Bhutan, BT, BTN)
+
+    def test_no_holidays(self):
+        self.assertNoHolidays(Bhutan(years=1969))
+
+    def test_winter_solstice(self):
+        self.assertHolidayName("Winter Solstice", (f"{year}-01-02" for year in range(1970, 2050)))
+
+    def test_traditional_day_of_offering(self):
+        name = "Traditional Day of Offering"
+        dt = ("2020-01-25", "2021-01-14", "2022-02-02", "2023-01-22", "2024-01-12", "2025-01-30")
+        self.assertHolidayName(name, dt)
```

Comment:
Same goes for other cases:
```suggestion
        self.assertHolidayName(
            name,
            "2020-01-25",
            "2021-01-14",
            "2022-02-02",
            "2023-01-22",
            "2024-01-12",
            "2025-01-30",
        )
        self.assertHolidayName(name, range(1970, 2050))
```